### PR TITLE
Feat homepage posts

### PR DESCRIPTION
**Pull request by rahulyadav139**

### CODE REVIEW

File: src/components/add-new-post/AddNewPostModal.js [hunk 1]
```diff
@@ -12,10 +12,9 @@ const AddNewPostModal = props => {
           <i class="fas fa-image"></i>
         </label>
         <img src="" alt="" />
```
Comment (mehulsatardekar) [inline src/components/add-new-post/AddNewPostModal.js]: remove this image tag or add some source to it.

File: src/App.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ import { useSelector } from 'react-redux';
 
 function App() {
   const isAuth = useSelector(state => state.auth.isAuth);
+  console.log('top');
```
Comment (junaidshaikh-js) [inline src/App.js:15]: remove console logs

**Review by mehulsatardekar — APPROVED** [review]
Rest things look good

File: src/components/add-new-post/AddNewPostModal.js [hunk 3]
```diff
@@ -1,19 +1,74 @@
...
+    const caption = captionRef.current.value;
+
+    const form = new FormData();
+
+    form.append('image', image);
+    form.append('caption', caption);
+
+    // const { error } = await sendData(
+    //   process.env.REACT_APP_BACKEND_URL + '/stats',
+    //   'POST',
```
Comment (mehulsatardekar) [inline src/components/add-new-post/AddNewPostModal.js]: remove commented code

**Review by mehulsatardekar — APPROVED** [review]

**Review by junaidshaikh-js — APPROVED** [review]
Good work. 

Clean code, liked the use of redux. Can be merged.

File: src/pages/homepage/Homepage.js [hunk 4]
```diff
@@ -8,43 +8,56 @@ import {
...
 
   useEffect(() => {
+    console.log('useEffect');
     (async () => {
       const { data, error, status } = await getData(
         process.env.REACT_APP_BACKEND_URL + '/post/all',
         true
       );
 
+      console.log(data.posts);
```
Comment (mehulsatardekar) [inline src/pages/homepage/Homepage.js:32]: remove logs

**Review by mehulsatardekar — APPROVED** [review]

**Pull request merged**